nuxeo-features/nuxeo-platform-imaging-tiling/nuxeo-platform-imaging-tiling-preview/src/main/java/org/nuxeo/ecm/platform/pictures/tiles/service/TiledImagePreviewer.java

```diff
@@ -88,8 +89,9 @@ protected boolean useTiling(Blob blob) {
             if (info != null) {
                 int width = info.getWidth();
                 int height = info.getHeight();
-                Integer widthThreshold = Integer.valueOf(PictureTilingComponent.getEnvValue("WidthThreshold", "1200"));
-                Integer heightThreshold = Integer.valueOf(PictureTilingComponent.getEnvValue("HeightThreshold", "1200"));
+                PictureTilingComponent ptc = (PictureTilingComponent) Framework.getService(PictureTilingService.class);
+                Integer widthThreshold = Integer.valueOf(ptc.getEnvValue("WidthThreshold", "1200"));
```

Comment:
shouldn't be a constant?

Reply:
The goal of this commit was to stop using the static methods of `PictureTilingComponent`. No other refactoring was done.